iron-iconset-svg.html

```diff
@@ -80,6 +80,16 @@
       rtlMirroring: {
         type: Boolean,
         value: false
+      },
+
+      /**
+       * Set to true to measure RTL based on the dir attribute on the body or
+       * html elements (measured on document.body or document.documentElement as
+       * available).
+       */
+      useGlobalRtlAttribute: {
```

Comment:
uhhh maybe RTL to match `__targetIsRT`? 🚲🏡

Reply:
Can you spell that as an attribute for me? :)